device-types/DataDirect Networks/SFA200NVX-HDR100.yaml

```diff
@@ -0,0 +1,75 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NVX-HDR100
+slug: sfa200nvx-hdr100
```

Comment:
All slugs should have the manufacturer appended. In this case, please append `datadirect-networks-`